src/modules/limitOrders/containers/LimitOrdersWarnings/index.tsx

```diff
@@ -60,25 +59,38 @@ export function LimitOrdersWarnings(props: LimitOrdersWarningsProps) {
   const { expertMode } = useAtomValue(limitOrdersSettingsAtom)
 
   const formState = useLimitOrdersFormState()
+  const tradeFormValidation = useGetTradeFormValidation()
   const rateImpact = useRateImpact()
   const { chainId, account } = useWalletInfo()
   const { slippageAdjustedSellAmount, inputCurrency, inputCurrencyAmount, outputCurrency, outputCurrencyAmount } =
     useLimitOrdersDerivedState()
+  const tradeQuote = useTradeQuote()
 
+  const canTrade = formState === null && tradeFormValidation === null && !tradeQuote.error
```

Comment:
`canTrade` when `formState === null`?
I would expect the inverse

Reply:
Sorry, it should have another name:
`formState ` -> `localFormValidation`

When a `validation === null` it means there are no validation errors or requests